homeassistant/components/camera/arlo.py

```diff
@@ -153,3 +153,21 @@ def disable_motion_detection(self):
         """Disable the motion detection in base station (Disarm)."""
         self._motion_status = False
         self.set_base_station_mode(ARLO_MODE_DISARMED)
+
+    @asyncio.coroutine
+    def async_update(self):
```

Comment:
make that only `update` without async and to that from `update_attributes`

Reply:
Re-examining this. When I make your suggested change, it seems to really slow things down (several cameras never get their attributes updated). With my original method (a coroutine), everything works. Thoughts?